app/models/ability.rb

```diff
@@ -167,7 +167,7 @@ def initialize(params)
             can :manage, UserBatch
 
             # can destroy users only if they have only one mission and it's the current mission
-            can :destroy, User do |other_user|
+            can [:bulk_destroy, :destroy], User do |other_user|
```

Comment:
Not sure if we need a separate ability for bulk_destroy. What is your reasoning here?